tests/codemods/test_fix_deprecated_logging_warn.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+from core_codemods.fix_deprecated_logging_warn import FixDeprecatedLoggingWarn
+from tests.codemods.base_codemod_test import BaseSemgrepCodemodTest
+
+
+class TestFixDeprecatedLoggingWarn(BaseSemgrepCodemodTest):
```

Comment:
It would be good to have another test with a call to `warn` that includes additional arguments just to make sure we preserve those arguments in the rewrite.
```python
logger.warn("something %s and %s", foo, bar)
```

Also I'm curious whether this works in this case:
```python
import logger
log = logging.getLogger('foo')

def some_function(logger):
    logger.warn("hi")
    
some_function(log)
```

It might be good to add the test and we can just xfail it for now if it doesn't work.